### PR TITLE
Hirak/save

### PR DESCRIPTION
**Pull request by Hirak20**
Fixing https://github.com/wtcounter/wtcounter/issues/8

### CODE REVIEW

File: app/src/main/java/wordtextcounter/details/main/feature/input/InputFragment.kt [hunk 1]
```diff
@@ -59,22 +57,20 @@ class InputFragment : BaseFragment() {
...
     RxBus.instance.send(ToolbarTitle(R.string.title_input))
     slidingUpPanelLayout.panelHeight = 0
     viewModel.viewState.observe(this, Observer {
       it?.let { it1 -> handleViewState(it1) }
     })
 
-    viewModel.reportState.observe(this, Observer {
-      it?.let { it1 -> handleReportState(it1) }
-    })
+
```
Comment (Apsaliya) [inline app/src/main/java/wordtextcounter/details/main/feature/input/InputFragment.kt]: delete this?

File: app/src/main/java/wordtextcounter/details/main/feature/input/InputViewModel.kt [hunk 2]
```diff
@@ -47,7 +47,9 @@ class InputViewModel : BaseViewModel() {
     reports.add(Report(Helper.countWords(input).toString(), WORDS))
     reports.add(Report(Helper.countParagraphs(input).toString(), PARAGRAPHS))
     reports.add(Report(Helper.calculateSize(input), SIZE))
-    reportState.value = ReportState(reports)
+
+    RxBus.instance.send(ReportState(reports))
+
```
Comment (Apsaliya) [inline app/src/main/java/wordtextcounter/details/main/feature/input/InputViewModel.kt]: and this?

File: app/src/main/java/wordtextcounter/details/main/feature/main/MainActivity.kt [hunk 3]
```diff
@@ -38,22 +38,22 @@ class MainActivity : BaseActivity() {
...
           toolbar.setTitle(it.title)
         })
 
+    bottombar.setOnTabSelectListener { tabId ->
+      when (tabId) {
+        R.id.tab_input -> replaceFragment(InputFragment.newInstance())
+        R.id.tab_notes -> replaceFragment(NotesFragment.newInstance())
+      }
+    }
+
```
Comment (Apsaliya) [inline app/src/main/java/wordtextcounter/details/main/feature/main/MainActivity.kt]: and this?

File: app/src/main/java/wordtextcounter/details/main/feature/notes/NotesViewModel.kt [hunk 4]
```diff
@@ -0,0 +1,14 @@
+package wordtextcounter.details.main.feature.notes
+
+import wordtextcounter.details.main.feature.base.BaseViewModel
+
+/**
+ * Created by hirak on 13/03/18.
+ */
```
Comment (Apsaliya) [inline app/src/main/java/wordtextcounter/details/main/feature/notes/NotesViewModel.kt]: I'm not sure we should include authors at the top of the files. I removed all yesterday. Let's not include these tags.

Comment (Hirak20) [reply app/src/main/java/wordtextcounter/details/main/feature/notes/NotesViewModel.kt]: Will do

**Pull request closed without merging**